content/webapp/components/Outro/Outro.tsx

```diff
@@ -101,6 +101,7 @@ const Outro = ({
                   });
                 }}
               >
+                OUTRO
```

Comment:
?

Reply:
Oops! 🙈 

You see some of my debugging code poking through.